src/Microsoft.AspNet.Mvc.Core/ActionResults/PartialViewResult.cs

```diff
@@ -80,7 +81,13 @@ public class PartialViewResult : ActionResult
 
             using (view as IDisposable)
             {
-                await ViewExecutor.ExecuteAsync(view, context, ViewData, TempData, ContentType);
+                await ViewExecutor.ExecuteAsync(
+                    view,
+                    context,
+                    ViewData,
+                    TempData,
+                    new HtmlHelperOptions(),
+                    ContentType);
```

Comment:
this means any `MvcOptions` changes made are irrelevant when executing the view. need to either pass the `HtmlHelperOptions` into this method or include the whole `MvcOptions` in the `ActionContext` when _that_ is created.